src/Sulu/Bundle/SecurityBundle/Command/CreateRoleCommand.php

```diff
@@ -43,13 +43,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $doctrine = $this->getContainer()->get('doctrine');
         $em = $doctrine->getManager();
-        $now = new \Datetime();
+        $name = $input->getArgument('name');
+        $system = $input->getArgument('system');
+
+        $repository = $em->getRepository('SuluSecurityBundle:Role');
+
+        $role = $repository->findOneByName($name);
+
+        if ($role) {
```

Comment:
I think that should happen directly after the initialization of `$name`. Just in case there will be more fields to fill in it would be quite annoying to get the error in the very end, and you have to reenter everything again.